File: binance4j-blvt/src/main/java/com/binance4j/blvt/dto/RedemptionResponse.java
package com.binance4j.blvt.dto;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;

/**
 * BLVT redemption response.
 * 
 * @param id           Subscription id.
 * @param status       Subscription status. S("success"), P("pending"), and F("failure").
 * @param tokenName    Token name.
 * @param amount       Redemption value in usdt.
 * @param redeemAmount Redemption token amount.
 * @param timestamp    Timestamp in ms.
 */
@ApiModel("BLVT redemption response.")
public record RedemptionResponse(@ApiModelProperty("Subscription id.") long id,
		@ApiModelProperty("Subscription status. S(\"success\"), P(\"pending\"), and F(\"failure\").") String status,
		@ApiModelProperty("Token name.") String tokenName, @ApiModelProperty("Redemption value in usdt.") String redeemAmount,
		@ApiModelProperty("Redemption token amount.") String amount, @ApiModelProperty("Timestamp in ms.") long timestamp) {
}

File: binance4j-blvt/src/main/kotlin/com/binance4j/blvt/dto/RedemptionResponse.kt
package com.binance4j.blvt.dto

import io.swagger.annotations.ApiModel
import io.swagger.annotations.ApiModelProperty

/**
 * BLVT redemption response.
 *
 * @property id           Subscription id.
 * @property status       Subscription status. S("success"), P("pending"), and F("failure").
 * @property tokenName    Token name.
 * @property amount       Redemption value in usdt.
 * @property redeemAmount Redemption token amount.
 * @property timestamp    Timestamp in ms.
 */
@ApiModel("BLVT redemption response.")
data class RedemptionResponse(
    @ApiModelProperty("Subscription id.")
    var id: Long? = null,
    @ApiModelProperty("Subscription status. S(\"success\"), P(\"pending\"), and F(\"failure\").")
    var status: String? = null,
    @ApiModelProperty("Token name.")
    var tokenName: String? = null,
    @ApiModelProperty("Redemption value in usdt.")
    var redeemAmount: String? = null,
    @ApiModelProperty("Redemption token amount.")
    var amount: String? = null,
    @ApiModelProperty("Timestamp in ms.")
    var timestamp: Long? = null
)